Resources/Prototypes/Entities/Objects/Misc/material_stacks.yml

```diff
@@ -155,6 +155,12 @@
     state: goldbar_10
   - type: Stack
     count: 1
+  - visuals:
+    - type: StackVisualizer
+      stackLayers:
+        - goldbar_10
+        - goldbar_20
+        - goldbar_30
```

Comment:
Should be on GoldStack given that's its parent? Also should have sprite netsync to false while you're in there plz and ty.